TestProject/Helper/UITextField+Extensions.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  Extension.swift
+//  TestProject
+//
+//  Created by propelrr-mac-pro on 6/13/22.
+//
+
+import UIKit.UITextField
+
+extension UITextField {
+    func validatedText(validationType: ValidatorType) throws -> String {
+        let validator = ValidatorFactory.validatorFor(type: validationType)
+        return try validator.validated(self.text!)
+    }
+}
+
+extension UITextField {
+    func togglePasswordVisibility() {
```

Comment:
You could move this function after the `validatedText` function since they belong to a single extension. For eg.:
![Screen Shot 2022-06-14 at 8 06 29 AM](https://user-images.githubusercontent.com/41320856/173466925-642c17fa-db02-4b94-8991-44f3470dce5d.png)